src/main/java/org/mifos/connector/slcb/dto/Transaction.java

```diff
@@ -4,7 +4,11 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.Getter;
+import lombok.Setter;
 
+@Getter
+@Setter
```

Comment:
how did this work with NoArg Constructor

Reply:
good question, Ill see into this.

Reply:
WIll solve in next PRs.